jupyterhub/values.yaml

```diff
@@ -157,3 +164,5 @@ cull:
   timeout: 3600
   every: 600
 
+pullBlocker:
+  image: {name: yuvipanda/k8s-pull-blocker, tag: 2b02899}
```

Comment:
why use inline dict here, rather than yaml-style like everywhere else?

Reply:
ah, pullBlocker is no longer used, this was a quick hack from earlier commits. I'll remove.